### PR TITLE
Image Template Update for group, quickfact, etc ... image

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator : _several tickets, checkout the below link_

### Problem Statement

 - [x] [T253224](https://phabricator.wikimedia.org/T253224) Group Image
 - [x] [T271697](https://phabricator.wikimedia.org/T271697) Quick fact image
 - [x] [T247735](https://phabricator.wikimedia.org/T247735) Pencil Icon
 - [x] Cleanup
 
Addition Fix not covered in above ticket
 - [x] Hide infobox in `fr/Chat` article

Another PR for
 - [T267634](https://phabricator.wikimedia.org/T267634) Label of Map
 
and I would like to have you to review it the changes code

### Solution

--
--

### CODE REVIEW

**Comment by hueitan** [comment]
what i notice recently all the image that also in gallery has this `<a>` tag with class name `image`

```html
<a href="/wiki/File:Cat_poster_1.jpg" class="image">
```

I will follow this approach and check more cases.

File: src/hooks/useArticleLinksNavigation.js [hunk 1]
```diff
@@ -13,6 +13,8 @@ const SUPPORTED_LINKS = [
   'a[href^="#"]',
   'figure',
   'figure-inline',
+  'div.tsingle',
```
[the remark this answers is not in the document]
Comment (hueitan) [reply src/hooks/useArticleLinksNavigation.js:16]: This is for the group image, `tsingle` includes `thumbimage` `thumbcaption` elements

```
<div class="thumb">
    <div class="thumbinner">
        <div class="trow">
            <div class="tsingle">
                <div class="thumbimage">
                    <div class="thumbcaption"></div>
                </div>
            </div>
        </div>
        <div class="trow">
            <div class="tsingle">
                <div class="thumbimage">
                    <div class="thumbcaption"></div>
                </div>
            </div>
        </div>
    </div>
</div>
```

[the remark this answers is not in the document]
Comment (stephanebisson) [reply src/hooks/useArticleLinksNavigation.js:16]: Did you check several wikis to see if it's the same?

[the remark this answers is not in the document]
Comment (hueitan) [reply src/hooks/useArticleLinksNavigation.js:16]: oh no! right, i see there are another form of group image in a different language.

[the remark this answers is not in the document]
Comment (hueitan) [reply src/hooks/useArticleLinksNavigation.js:16]: just fix another one found template

```html
<ul class="gallery">
    <li class="gallerybox">
        <div class="thumb"></div>
        <div class="gallerytext"></div>
    </li>
    <li class="gallerybox">
        <div class="thumb"></div>
        <div class="gallerytext"></div>
    </li>
</ul>
```

let me know if you see other cases (definitely not cover 100% yet)

**Review by stephanebisson — APPROVED** [review]
I feel that chasing all templates and styling them correctly is a lost battle... but this seems to help.

**Pull request merged**